style.css

```diff
@@ -104,6 +104,222 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  width: 100%;
+  height: 2788px;
+  top: 718px;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 16px, 6px;
+  border-radius: 16px;
+  margin: 36px;
+  border: 1px solid #dfe1e6;
+}
+
+.cards {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+  padding-left: 24px;
+}
+
+.snapshot-portofolio img {
+  width: 295px;
+  height: 220px;
+}
+
+.blocks {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.txtWrapper {
+  width: 80%;
+}
+
+.project-title {
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #344563;
+  padding: 5px;
+}
+
```

Comment:
- [ ] Please make sure to use the same color as given in the Figma design for title 🛠 In Figma design guidelines, it is `#172B4D`